drafts/2019-11-06-1900011.adoc

```diff
@@ -157,6 +160,95 @@ MetadataBuilder mdb = Metadata.builder();
 mdb = mdb.withName("metricName").withType(MetricType.COUNTER)
   .reusable(resolveIsReusable());
 ----
+[#JMO]
+== Jaeger for MicroProfile OpenTracing
+
+OpenLiberty can use Jaeger as a tracing backend for MicroProfile OpenTracing. 
+
+OpenLiberty has the support of MicroProfile OpenTracing. A sample tracer was provided at link:https://github.com/WASdev/sample.opentracing.zipkintracer[Opentracer] for using Zipkin as a tracing backend. With the introduction of jaeger-1.0, developers can also use Jaeger as a tracing backend.
+
+You will need to download the Jaeger client version 0.34.0 library and its dependencies from link:https://mvnrepository.com/artifact/io.jaegertracing/jaeger-client/0.34.0[Maven repository].
+
+In the server.xml 
```

Comment:
Add a colon (:) after the lead-in. So "In the `server.xml`:"

Then tidy up the capitalisation, highlighting, and punctuation of the two steps:

"Add `<feature>jaeger-1.0</feature>` in the `<featureManager>` section:"

"Define your application:"

Reply:
Also, can you get the left alignment of the XML to line up in the syntax highlighted snippets? eg the webApplication open and closing elements aren't aligned. Just looks messy.